hazelcast/src/main/java/com/hazelcast/util/VersionCheck.java

```diff
@@ -154,13 +145,9 @@ private Document fetchWebService(String urlStr) {
             conn.setConnectTimeout(TIMEOUT * 2);
             conn.setReadTimeout(TIMEOUT * 2);
             in = new BufferedInputStream(conn.getInputStream());
-            final DocumentBuilder builder = DocumentBuilderFactory.newInstance().newDocumentBuilder();
-            return builder.parse(in);
-        } catch (SAXException ignored) {
-            EmptyStatement.ignore(ignored);
-        } catch (IOException ignored) {
-            EmptyStatement.ignore(ignored);
-        } catch (ParserConfigurationException ignored) {
+            IOUtil.closeResource(in);
+            return null;
```

Comment:
I don't think return value is used by anyone so we can change the return type of the method to ```void```